### PR TITLE
Add tests to check re-share behavior inside blacklisted from sharing group

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
This PR webUI tests for files_external delete local storage mount.

Related issue:  #33824

## How Has This Been Tested?
🤖

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature [hunk 1]
```diff
@@ -108,18 +108,56 @@ Feature: Sharing files and folders with internal groups
...
+    And the administrator has enabled exclude groups from sharing from admin sharing settings page
+    When the administrator adds group "grp1" to the group sharing blacklist using the webUI
+    Then user "user1" should not be able to share file "/common/testimage.jpg" with user "user4" using the sharing API
+
+  Scenario: member of a blacklisted from sharing group tries to re-share a folder inside a folder received as a share
+    Given user "user4" has been created with default attributes
+    And the administrator has browsed to the admin sharing settings page
+    And user "user3" has created folder "/common"
+    And user "user3" has created folder "/inside-common"
+    And user "user3" has moved file "/inside-common" to "/common/inside-common"
```
[the remark this answers is not in the document]
Comment (paurakhsharma) [reply tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature]: Yeah, thanks

**Review by phil-davis — DISMISSED** [review]

**Comment by paurakhsharma** [comment]
Backport on: #33866

**Review by skshetry — DISMISSED** [review]

**Review by skshetry — APPROVED** [review]

File: tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature [hunk 2]
```diff
@@ -103,16 +108,119 @@ Feature: Sharing files and folders with internal users
...
+    And user "user3" has been created with default attributes
+    And the administrator has browsed to the admin sharing settings page
+    And the administrator has enabled exclude groups from sharing from admin sharing settings page
+    When the administrator adds group "grp1" to the group sharing blacklist using the webUI
+    And the user re-logs in as "user1" using the webUI
+    And the user opens sharing tab from file action menu of file "testimage.jpg" using the webUI
+    Then the user should see a error message on the share dialog saying "Sharing is not allowed"
+    And the share-with field should not be visible in the details panel
+
+  @enterprise-issue-3037 # after the issue is fixed the issue remove the comments on two lines and remove last line
```
Comment (phil-davis) [inline tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature]: s/after the issue is fixed the issue remove/after the issue is fixed remove

**Review by phil-davis — COMMENTED** [review]
Looks good, just some comments on naming

**Review by phil-davis — APPROVED** [review]

**Pull request merged**